c10/core/ScalarType.h

```diff
@@ -10,6 +11,22 @@
 
 namespace c10 {
 
+// NB: Order matters for this macro; it is relied upon in
+// _promoteTypesLookup and the serialization format.
+#define AT_FORALL_SCALAR_TYPES_WITH_COMPLEX_QUANTIZE(_) \
+_(uint8_t,Byte,i)  /* 0 */ \
+_(int8_t,Char,i)   /* 1 */ \
+_(int16_t,Short,i) /* 2 */ \
+_(int,Int,i)       /* 3 */ \
+_(int64_t,Long,i)  /* 4 */ \
+_(at::Half,Half,d) /* 5 */ \
+_(float,Float,d)   /* 6 */ \
+_(double,Double,d) /* 7 */ \
+_(at::ComplexHalf,ComplexHalf,z)        /* 8 */ \
+_(std::complex<float>,ComplexFloat,z)   /* 9 */ \
+_(std::complex<double>,ComplexDouble,z) /* 10 */ \
+_(QInt,QInt,i) /* 11 */
```

Comment:
wait, it doesn't make any sense - this is the type of value in the struct, QInt seems to be some type descriptor, not value type (which should be uint8 probably)